src/sass/components/_page-footer.scss

```diff
@@ -0,0 +1,167 @@
+            margin-left: auto;
+            padding-top: 40px;
+            padding-bottom: 40px;
+        }
+    
+        @media screen and (min-width: 1280px) {
+            justify-content: start;
+            padding-top: 52px;
+            padding-bottom: 53px;
+        }
+    }
+
+    &__phone {
+        display: inline-flex;
+        align-items: center;
+        margin-top: 20px;
+        font-family: var(--main-font-name);
+        text-decoration: none;
+        font-weight: 600;
+        font-size: 14px;
+        line-height: 1.2;
+        letter-spacing: 0.08;
+        color: var(--secondary-bg-color);
+
+        @media screen and (min-width: 768px) {
+            margin-top: 0;
+            margin-right: 24px;
+            margin-left: auto;        
+            padding-right: 24px;
+            line-height: 1.7;
+            border-right: 1px solid var(--third-txt-color);
+            font-weight: 600;
+            font-size: 20px;
+        }
+
+        @media screen and (min-width: 1280px) {
+            margin-left: 290px;
+            font-size: 30px;
+            line-height: 1.3;
+        }
+    }
+
+    &__icon {
+        fill: var(--accent-color);
+        margin-right: 15px;
+
+        @media screen and (max-width: 767px) {
+            width: 20px;
+            height: 19px;
+        }
+
+        @media screen and (max-width: 1279px) {
+            width: 24px;
+            height: 23px;
+            margin-right: 10px;
+        }
+    }
+
+    &__instagram {
+        margin-top: 30px;
+        display: inline-block;
+        width: 40px;
+        height: 40px;
+        background-color: var(--main-bg-color);
+        background-image: url(../images/instagram-logo.svg);
+        transition: transform 250ms var(--time-finction);
+
+        @media screen and (min-width: 768px) {
+            justify-content: right;
+            margin-top: 0;
+        }
+    }
+
+    &__instagram:hover,
+    &__instagram:focus {
+        transform: scale(1.1) ;
+    }
+}
+
+.logo-mobile {
+    @media screen and (min-width: 768px) {
+        display: none;
+    }
+}
+
+.logo {
+    &__name {
+        text-align: center;
+        display: inline-flex;
+        align-items: center;
+    }
+
+    &__name-en {
+        font-family: var(--main-font-name);
+        font-weight: 800;
+        font-size: 26px;
+        line-height: 1.23;
+        letter-spacing: 0.04em;
+        color: var(--secondary-bg-color);
+    }
+
+    &__icon {
+        fill: var(--accent-color);
+        margin-right: 5px;
+        margin-bottom: 5px;
+        margin-left: 5px;
+    }
+
+    &__name-ru {
+        font-family: var(--main-font-name);
+        font-weight: 700;
+        font-size: 8px;
+        line-height: 1.25;
+        letter-spacing: 0.18em;
+        text-transform: uppercase;
+        text-align: center;
+        color: var(--secondary-bg-color);
+    }
+
+    @media screen and (min-width: 768px) {
+        &__name-accent {
```

Comment:
.logo__name-accent { 
@media screen and (min-width: 768px) {
                  color: var(--accent-color);
        }
    }